addon/exports/service.js

```diff
@@ -117,8 +117,16 @@ export default Service.extend({
       .then(callback);
   },
 
-  searchEntities(keyword) {
-    return this.ajax.request(`${this._exportURL}/entities?s=${encodeURIComponent(keyword)}`, {
+  searchEntities(keyword, entityTypes = []) {
+    const params = {
+      s: encodeURIComponent(keyword)
+    };
+
+    if (entityTypes.length > 0) {
+      params['entity_types'] = entityTypes.join(',');
+    }
+
+    return this.ajax.request(`${this._exportURL}/entities?${new URLSearchParams(Object.entries(params)).toString()}`, {
```

Comment:
Continue to use ajax?

Reply:
just didn't want to touch this file too much, plus it will be moved and refactored in ember-influencer at some point